ast panic : Inherited

```
panic: could not match regexp with string
^(?P<address>[0-9a-fx]+) <(?P<position>.*)>(?P<a> \d+)(?P<b> \d+)?(?P<c> \d+)?(?P<d> \d+)?[\s]*$
0x39cf2b0 <col:53> Inherited 0 1


goroutine 16 [running]:
github.com/elliotchance/c2go/ast.groupsFromRegex(0xc422036cc0, 0x60, 0xc4219fade0, 0x20, 0x2)
	/home/konstantin/go/src/github.com/elliotchance/c2go/ast/ast.go:250 +0x3cc
github.com/elliotchance/c2go/ast.parseNonNullAttr(0xc4219fade0, 0x20, 0x5e83c7)
	/home/konstantin/go/src/github.com/elliotchance/c2go/ast/non_null_attr.go:23 +0x62
github.com/elliotchance/c2go/ast.Parse(0xc4219fadd4, 0x2c, 0x5e6cca, 0x5)
	/home/konstantin/go/src/github.com/elliotchance/c2go/ast/ast.go:166 +0x25c8
main.convertLinesToNodes(0xc420579d40, 0x489d, 0x489d, 0xc420204010, 0x5a8100, 0xc42006c1e0)
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:96 +0x196
main.convertLinesToNodesParallel.func1.2(0xc42006c2a0, 0xc42000c008, 0xc420579d40, 0x489d, 0x489d, 0x0)
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:128 +0x59
created by main.convertLinesToNodesParallel.func1
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:132 +0x1b6
```

--- a/ast/non_null_attr.go
+++ b/ast/non_null_attr.go
@@ -9,6 +9,7 @@ import (
 type NonNullAttr struct {
 	Addr       Address
 	Pos        Position
+	Inherited  bool
 	A          int
 	B          int
 	C          int
@@ -18,7 +19,9 @@ type NonNullAttr struct {
 
 func parseNonNullAttr(line string) *NonNullAttr {
 	groups := groupsFromRegex(
-		`<(?P<position>.*)>(?P<a> \d+)(?P<b> \d+)?(?P<c> \d+)?(?P<d> \d+)?`,
+		`<(?P<position>.*)>
+		(?P<inherited> Inherited)?
+		(?P<a> \d+)(?P<b> \d+)?(?P<c> \d+)?(?P<d> \d+)?`,
 		line,
 	)
 
@@ -40,6 +43,7 @@ func parseNonNullAttr(line string) *NonNullAttr {
 	return &NonNullAttr{
 		Addr:       ParseAddress(groups["address"]),
 		Pos:        NewPositionFromString(groups["position"]),
+		Inherited:  len(groups["inherited"]) > 0,
 		A:          util.Atoi(strings.TrimSpace(groups["a"])),
 		B:          b,
 		C:          c,

--- a/ast/non_null_attr_test.go
+++ b/ast/non_null_attr_test.go
@@ -9,6 +9,7 @@ func TestNonNullAttr(t *testing.T) {
 		`0x7fa1488273b0 <line:7:4, line:11:4> 1`: &NonNullAttr{
 			Addr:       0x7fa1488273b0,
 			Pos:        NewPositionFromString("line:7:4, line:11:4"),
+			Inherited:  false,
 			A:          1,
 			B:          0,
 			C:          0,
@@ -18,6 +19,7 @@ func TestNonNullAttr(t *testing.T) {
 		`0x2cce280 </sys/cdefs.h:286:44, /bits/mathcalls.h:115:69> 1`: &NonNullAttr{
 			Addr:       0x2cce280,
 			Pos:        NewPositionFromString("/sys/cdefs.h:286:44, /bits/mathcalls.h:115:69"),
+			Inherited:  false,
 			A:          1,
 			B:          0,
 			C:          0,
@@ -27,6 +29,7 @@ func TestNonNullAttr(t *testing.T) {
 		`0x201ede0 <line:145:79, col:93> 0`: &NonNullAttr{
 			Addr:       0x201ede0,
 			Pos:        NewPositionFromString("line:145:79, col:93"),
+			Inherited:  false,
 			A:          0,
 			B:          0,
 			C:          0,
@@ -36,6 +39,7 @@ func TestNonNullAttr(t *testing.T) {
 		`0x1b89b20 <col:76, col:93> 2 3`: &NonNullAttr{
 			Addr:       0x1b89b20,
 			Pos:        NewPositionFromString("col:76, col:93"),
+			Inherited:  false,
 			A:          2,
 			B:          3,
 			C:          0,
@@ -45,6 +49,7 @@ func TestNonNullAttr(t *testing.T) {
 		`0x55f0219e20d0 <line:717:22, col:42> 0 1 4`: &NonNullAttr{
 			Addr:       0x55f0219e20d0,
 			Pos:        NewPositionFromString("line:717:22, col:42"),
+			Inherited:  false,
 			A:          0,
 			B:          1,
 			C:          4,
@@ -54,10 +59,21 @@ func TestNonNullAttr(t *testing.T) {
 		`0x248ea60 <line:155:26, col:49> 0 1 2 4`: &NonNullAttr{
 			Addr:       0x248ea60,
 			Pos:        NewPositionFromString("line:155:26, col:49"),
+			Inherited:  false,
 			A:          0,
 			B:          1,
 			C:          2,
 			D:          4,
+			ChildNodes: []Node{},
+		},
+		`0x39cf2b0 <col:53> Inherited 0 1`: &NonNullAttr{
+			Addr:       0x39cf2b0,
+			Pos:        NewPositionFromString("col:53"),
+			Inherited:  true,
+			A:          0,
+			B:          1,
+			C:          0,
+			D:          0,
 			ChildNodes: []Node{},
 		},
 	}

